src/frontend/src/lib/derived/tokens.derived.ts

```diff
@@ -55,3 +57,20 @@ export const enabledIcTokens: Readable<IcToken[]> = derived(
 	([$enabledTokens]) =>
 		$enabledTokens.filter(({ standard }) => standard === 'icp' || standard === 'icrc') as IcToken[]
 );
+
+export const enabledMainnetErc20TokensTotalUsd: Readable<number> = derived(
```

Comment:
Does the use case require different derived stores, or is the information used for the same feature? If it's the latter, it would be more efficient to iterate over the tokens once to calculate the sum.

Reply:
I'd say they have to be implemented separately, as all 3 stores will be used in the following PR where I add UI components for this feature.

Reply:
Actually, regardless of how many derived stores you need, would it be more efficient to derive an object containing the sums - e.g. `{total: number, ic: number, erc20: number}` - by iterating over the tokens only once. Afterward, whether you use the entire object directly or access specific properties, the performance cost will be minimal.

WDYT?

Reply:
This is a great idea 👍 This way, if I use network id as a key for this dictionary, I can easily access this data in `Network` component, without manually checking `isNetworkIdICP`, `isNetworkIdEthereum` and so on.

Reply:
Done. The store is now `Record<NetworkId, number>`. The total value can now be calculated just by adding all values.